### ISSUE TITLE
Detects only 20 objects.

### ISSUE THREAD

**Issue by Aswathi-Mampully** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#500]
I followed the exact process and training is going well. The loss has come really down to 0.09. But i noticed that only 20 objects are detected in the test image everytime. Is there anything i should change so as to get maximum number of objects?
